### DISCUSSION TITLE
Airsim drone dimensions

### DISCUSSION THREAD

**Discussion by krishnarajk** [microsoft/AirSim#4585] [General]
What is the default size of the drone in the Airsim - unreal engine. How can I change the size, length/width of the drone?

**Comment by ClosetGeek-Git** [comment]
+1 I think the docs should have a more emphasis on dimensions in general. Obviously different models will have different sizes, as will every tree and every hill. But I haven't really seen anything regarding dimensions in general terms.

**Comment by ClosetGeek-Git** [comment]
Maybe a feature that will allow you to click between two points and see the distance, almost like you can do with an iPhone. Even if it's just approximate.

**Comment by alonfaraj** [comment]
> What is the default size of the drone in the Airsim - unreal engine. How can I change the size, length/width of the drone?

@krishnarajk 
The default dimension is 1m*1m
You can change it the content browser:
`/AirSim/Blueprints/BP_FlyingPawn`
select `BodyMesh` from components tab -> `Transform` -> `scale`

**Reply by krishnarajk** [reply]
@alonfaraj thanks for the info

**Comment by zardosht** [comment]
This issue can also be related: https://github.com/microsoft/AirSim/issues/3172
